README.md

```diff
@@ -5,56 +5,65 @@ These are project templates that are used by Create React App when starting a ne
+
+The [Routing template](https://www.npmjs.com/package/@brightlayer-ui/cra-template-routing-typescript) provides all of the essential Brightlayer UI configuration as well as additional configuration for using using [React Router](https://reactrouter.com/). It also adds a navigation Drawer to switch between several sample screens.
 
 ### Authentication
-The Authentication template includes everything from the Routing template and also installs and configures the Brightlayer UI [react-auth-workflow](https://www.npmjs.com/package/@brightlayer-ui/react-auth-workflow) package to automatically wrap the main application with functions and screens for Login, Registration, Change Password, etc. This template is available in [TypeScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-authentication-typescript)(recommended) and [JavaScript](https://www.npmjs.com/package/@brightlayer-ui/cra-template-authentication).
 
-|                | placeholder landing page | navigation drawer & routing  | login & registration screens  |
-| -------------- | -- | -- | -- |
-| Blank          | ✅ |    |    |
-| Routing        | ✅ | ✅ |    |
-| Authentication | ✅ | ✅ | ✅ |
+The [Authentication template](https://www.npmjs.com/package/@brightlayer-ui/cra-template-authentication-typescript) includes everything from the Routing template and also installs and configures the Brightlayer UI [react-auth-workflow](https://www.npmjs.com/package/@brightlayer-ui/react-auth-workflow) package to automatically wrap the main application with functions and screens for Login, Registration, Change Password, etc.
 
+|                | placeholder landing page | navigation drawer & routing | login & registration screens |
+| -------------- | ------------------------ | --------------------------- | ---------------------------- |
+| Blank          | ✅                       |                             |                              |
+| Routing        | ✅                       | ✅                          |                              |
+| Authentication | ✅                       | ✅                          | ✅                           |
 
 ## Screenshots
+
 ##### Routing Template With Collapsed Drawer
+
 ![Routing Template With Collapsed Drawer](./images/routing.png)
+
 ##### Routing Template With Expanded Drawer
+
 ![Routing Template With Expanded Drawer](./images/routing-expanded.png)
+
 ##### Login Screen from Auth Template
+
 ![Authentication Template Login](./images/authentication.png)
 
 ## Usage
+
 You can use these templates with the Brightlayer UI CLI (recommended):
 
 ```sh
-npx -p @brightlayer-ui/cli blui new react --template=blank
```

Comment:
Are we still able to accept this shorthand template name, but use the typescript implementation of the template (i.e., template=blank still works, but uses the typescript template)?

Reply:
Yes we are still using javascript template's shorthand and create typescript template. If I provide --template=blank it creates blank-typescript project